src/components/Search/useSearchController.ts

```diff
@@ -48,5 +50,6 @@ export const useSearchController = (params: SearchControllerParams) => {
     handleSearch,
     handleResetSearchInput,
     handlePress,
+    t,
```

Comment:
This shouldn't be a part of the controller until you need to translate something inside the controller. In all other cases use useTranslation() directly in the component